sdk/core/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub mod range;
 mod stored_access_policy;
 pub mod util;
 
-use errors::AzureError;
+pub use errors::AzureError;
```

Comment:
Any reason why it wasn't pub?

Reply:
Not that I know of

Reply:
It is already exposed as `azure_core::errors::AzureError`.